BSseq_pipeline.py

```diff
@@ -490,8 +488,15 @@
     log:
         os.path.join(DIR_methcall,"{prefix}.sorted_meth_calls.log")
     message: fmt("Extract methylation calls from bam file.")
-    run:
-        generateReport(input, output, params, log, wildcards.prefix)
+    shell:
+        nice(" ".join("{RSCRIPT},"{DIR_scripts}/methCall.R",
```

Comment:
nice takes two or three arguments, so this should be:

    nice('Rscript', ["{DIR_scripts}/methCall.R",
                           "--inBam {params.inBam}",
                            ...
                           ])